### PR TITLE
Add cellPadFilter functions

### PR DESCRIPTION
**Pull request by Megamouse**
Adds some random functions and logging.

Idk if the functions are added properly.

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellPad.cpp [hunk 1]
```diff
@@ -972,6 +986,27 @@ error_code cellPadLddUnregisterController(s32 handle)
...
+	return CELL_OK;
+}
+
+u32 cellPadFilterIIRFilter(vm::ptr<CellPadFilterIIRSos> pSos, u32 filterIn)
+{
+	sys_io.todo("cellPadFilterIIRFilter(pSos=*0x%x, filterIn=%d)", pSos, filterIn);
+
+	// TODO: apply filter
+
+	return std::clamp(filterIn, 0u, 1023u);
```
Comment (elad335) [inline rpcs3/Emu/Cell/Modules/cellPad.cpp]:
```suggestion
	return std::clamp<u32>(filterIn, 0, 1023);
```

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellPad.cpp]: Is this in any way different apart from how it looks?

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellPad.cpp]: Its clearly more readable and doesn't look like a compilation issue workaround.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellPad.cpp]: 0 and 32 are passed as int though, whereas 0u and 1023u are clearly u32

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellPad.cpp]: yours looks more like a compilation workaround tbh

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellPad.cpp]: by that logic everything in cpp is a compilation workaround

**Pull request merged**